src/vstart.sh

```diff
@@ -754,7 +765,7 @@ EOF
 	    echo "{\"cephx_secret\": \"$OSD_SECRET\"}" > $CEPH_DEV_DIR/osd$osd/new.json
             ceph_adm osd new $uuid -i $CEPH_DEV_DIR/osd$osd/new.json
 	    rm $CEPH_DEV_DIR/osd$osd/new.json
-            $SUDO $CEPH_BIN/ceph-osd -i $osd $ARGS --mkfs --key $OSD_SECRET --osd-uuid $uuid
+            $SUDO $CEPH_BIN/$ceph_osd $crimson_args -i $osd $ARGS --mkfs --key $OSD_SECRET --osd-uuid $uuid
```

Comment:
s/crimson_args/extra_osd_args?

Seems like it could be used to pass anything to the osd.

Reply:
@athanatos fixed and repushed.